docs/tradeoffs.md

```diff
@@ -116,7 +119,7 @@ Both libraries provide derive APIs that mask over the static and dynamic typing
 
 In `bpaf`s case, the combinators still show through in terms of requiring the
 user to organize their data structures around their validation.  Some times
-this is good (pushing errors to compile time like if mutually exclusive
+this is good (pushing errors to design and compile time like if mutually exclusive
```

Comment:
"design time" here means if I know that something is `enum`  - I don't even consider cases where A and B are possible at the same time.

Reply:
I think the distinction with compile time is too minor to call out and will just be more confusing.

Reply:
Pretty significant at least to me personally, but sure, removed.